### PR TITLE
Add error_handlers

### PR DESCRIPTION
**Pull request by MingshiPeng**
1. Create a new python file for handling errors
1. Add error handler for 500 Internal Error for debugging Test stage errors.
1. change logging level for error handlers from "logging" to "error"

### CODE REVIEW

File: app/server.py [hunk 1]
```diff
@@ -6,7 +6,10 @@
 import logging
 import sys
 from app import app
-from app.models import DataValidationError, ProductInformation
+# Error handlers reuire app to be initialized so we must import
```
Comment (majia67) [inline app/server.py]: `reuire` should be `require`

**Review by majia67 — CHANGES_REQUESTED** [review]
Not sure the main motivation of this PR?

**Comment by MingshiPeng** [comment]
The reason for this PR is that I was constantly getting 500 errors during Test stage setup without knowing the error message. So I created the error handlers.

**Comment by majia67** [comment]
I don't think it's necessary to separate the error handlers into a separate file? This just makes code logic more complicated. I'll unblock you and leave the decision to yourself.

**Review by majia67 — APPROVED** [review]

**Comment by MingshiPeng** [comment]
For code coverage purposes because some of the error handlers won't be touched during Unit test.

**Comment by MingshiPeng** [comment]
Also make the server.py code cleaner.

**Comment by majia67** [comment]
IBM Cloud [toolchain](https://console.ng.bluemix.net/devops/toolchains/d4dd8efb-da17-4b0c-b4e6-5599ec73dd60?env_id=ibm%3Ayp%3Aus-south "nyu-inventory-service-s18"): [Delivery Pipeline](https://console.ng.bluemix.net/devops/pipelines/c71be43a-8963-4052-b97c-5da9b99de3f9?env_id=ibm%3Ayp%3Aus-south "nyu-inventory-service-s18") deployed [inventory](https://github.com/NYUDevOpsInventory/inventory/releases/tag/deploy-ibm-yp-us-south-NYU_Devops_Inventory_Org-dev-20180501-192921 "deploy-ibm-yp-us-south-NYU_Devops_Inventory_Org-dev-20180501-192921") to [dev](https://console.bluemix.net/apps/1db994d8-c4f1-4076-85c0-7ff2674244e5?env_id=ibm:yp:us-south "ibm:yp:us-south:NYU_Devops_Inventory_Org:dev"), including commit ed29652a5730b959bac89c58ffeb17749fc4718b

**Comment by majia67** [comment]
IBM Cloud [toolchain](https://console.ng.bluemix.net/devops/toolchains/d4dd8efb-da17-4b0c-b4e6-5599ec73dd60?env_id=ibm%3Ayp%3Aus-south "nyu-inventory-service-s18"): [Delivery Pipeline](https://console.ng.bluemix.net/devops/pipelines/c71be43a-8963-4052-b97c-5da9b99de3f9?env_id=ibm%3Ayp%3Aus-south "nyu-inventory-service-s18") deployed [inventory](https://github.com/NYUDevOpsInventory/inventory/releases/tag/deploy-ibm-yp-us-south-NYU_Devops_Inventory_Org-prod-20180501-193212 "deploy-ibm-yp-us-south-NYU_Devops_Inventory_Org-prod-20180501-193212") to [prod](https://console.bluemix.net/apps/e25e653b-a50f-4831-b93e-d8f9f484a484?env_id=ibm:yp:us-south "ibm:yp:us-south:NYU_Devops_Inventory_Org:prod"), including commit ed29652a5730b959bac89c58ffeb17749fc4718b

**Pull request merged**